File: 06376.py
print('n e')
print('- -----------')

n = 0; i_pac = 1; e = 1
for i in range(10):
    if n == 0 or n == 1:
        print("%d %d" % (n, e))
    elif n == 2:
        print("%d %.1f" % (n, e))
    else:
        print("%d %.9f" % (n, e))

    n += 1
    i_pac *= n
    e += (1 / i_pac)

File: 06376.cpp
#include <iostream>
using namespace std;

int main() {
	cout << "n e\n";
	cout << "- -----------\n";

	int n = 0, i_pac = 1;
	double e = 1;
	for (int i = 0; i < 10; i++) {
		if (n >= 3) {
			cout << fixed;
			cout.precision(9);
		}
		cout << n << ' ' << e << endl;

		n++;
		i_pac *= n;
		e += (1 / (double)i_pac);
	}
}
